tests/helpers/test_cap.c

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
This is just MY opinion.

This program is so useful and short. MIT license can be an option.

Reply:
Done, although I would hope anyone could implement these 20 lines of code :-)